src/ApiRequestPanel.js

```diff
@@ -683,11 +737,21 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     this.updateServers({ id: selected, type, endpointId });
   }
 
+  _handleServersCountChange(e) {
```

Comment:
Genuine question: Do we need to bind this function in the constructor if it is not used as a conventional event listener?

Reply:
https://lit-element.readthedocs.io/en/v0.6.4/docs/templates/databinding/#eventhandler